.github/workflows/commitlint.yml

```
@@ -13,7 +13,9 @@ jobs:
         with:
           fetch-depth: 0
           ref: ${{ github.event.pull_request.head.sha }}
+          path: pr_nvim
       - uses: rhysd/action-setup-vim@v1
         with:
           neovim: true
-      - run: nvim --clean -es +"lua require('scripts.lintcommit').main({trace=true})"
+      - run: wget https://github.com/neovim/neovim/blob/master/scripts/lintcommit.lua
+      - run: nvim --clean -es +"lua lint = require('lintcommit')" +"cd pr_nvim" +"lua lint.main({trace=true})"
```

```suggestion
      - run: nvim --clean -es +"lua lint = dofile('./lintcommit.lua')" +"cd pr_nvim" +"lua lint.main({trace=true})"
```
making sure it's not shadowed by other scripts in the package path